### ISSUE TITLE
Lib 1.1 not found on maven

### ISSUE THREAD

**Issue by nilson-aguiar** [sababado/CircularView#6]
Please check and upload it :D

**Comment by sababado** [comment]
Well that's unexpected. I've got my AAR on Bintray, linked to jCenter. I'm not sure why it hasn't been published.

**Comment by sababado** [comment]
I think I see the issue. I changed my publishing script the last time I uploaded the release version. I accidentally changed the artifact id. I'm going to make that change and republish. It should be available in a few hours.

**Comment by nilson-aguiar** [comment]
Thanks for the fast reply. I'll try again later!

**Comment by nilson-aguiar** [comment]
Error:Could not find library.jar (com.sababado.circularview:library:1.1.0).
Searched in the following locations:
    https://jcenter.bintray.com/com/sababado/circularview/library/1.1.0/library-1.1.0.jar

Still nothing :/

**Comment by sababado** [comment]
The script I have only uploaded a POM file -_-
Sorry about this, I'm not sure why all of a sudden I'm having these issues.

**Comment by sababado** [comment]
TL;DR -> I've modified my buildscript and pushed again. It 1.1.0 should be available as soon as it is processed.

I see my issue. For this version I changed the build script and how it pushes to bintray. I don't remember if the version prior to 1.1.0 also went directly to bintray. Anyhow, there was a convention I wasn't following, in that the module name has to be the exact name that will appear in the archive path. Which wasn't the case here. So I've pushed it and things are matching up a bit more now.
